apps/design-system-docs/docs/components/ds/feedback/index.mdx

```diff
@@ -0,0 +1,10 @@
+---
+sidebar_position: 3
+---
+
+import { LinkCardListFromMdx } from '@site/src/components/link-card-list';
```

Comment:
Nice that Docusaurus has these aliases for `@site/`, etc. I had proposed [subpath imports](https://nodejs.org/api/packages.html#subpath-imports) in https://github.com/Automattic/wp-calypso/pull/103561 not realizing it had built-in aliases. Subpath imports feel a little more Node.js-native and less vendor-locked, but I imagine the aliases are more conventional and well-supported in Docusaurus.

No strong preference, just figured I'd raise it as a discussion point.